trustpoint/home/tables.py

```diff
@@ -38,7 +38,7 @@ class Meta:
             'created_at',
             'details',
             #'solve',
-            'delete',
+            #'delete',
```

Comment:
Is it correct that deleting/dismissing notifications is currently not implemented and out of scope for the beta?

Reply:
I was not sure if deleting a notification is something we want to provide. The actual model also would recreate the same notification with the next run as it checks wether a notification for the issue already exists